src/Raven.Server/Documents/Handlers/Processors/AbstractHandlerProcessor.cs

```diff
@@ -43,10 +46,21 @@ internal abstract class AbstractDatabaseHandlerProcessor<TRequestHandler, TOpera
 
         protected readonly Logger Logger;
 
+        public AuthenticateConnection AuthenticateConnection;
+
+
         protected AbstractDatabaseHandlerProcessor([NotNull] TRequestHandler requestHandler) : base(requestHandler)
         {
             ContextPool = requestHandler.ContextPool;
             Logger = requestHandler.Logger;
+            AuthenticateConnection = HttpContext.Features.Get<IHttpAuthenticationFeature>() as AuthenticateConnection;
+        }
+
+        public AuthorizationStatus GetAuthorizationStatus(string databaseName)
```

Comment:
Same here

Reply:
Remove this and use it directly via RequestHandler.GetXYZ

Reply:
Done